src/main/java/domain/LineGenerator.java

```diff
@@ -0,0 +1,26 @@
+package domain;
+
+import java.security.SecureRandom;
+import java.util.ArrayList;
+import java.util.List;
+
+public class LineGenerator {
+    public Line generate(int personCount) {
+        final SecureRandom secureRandom = new SecureRandom();
```

Comment:
```suggestion
public class LineGenerator {

    private static final SecureRandom SECURE_RANDOM = new SecureRandom();
    
    public Line generate(int personCount) {
        
```
이런 객체들은 보통 static 변수로 관리해주는것 같아요
그리고 클래스 선언부와
필드변수 그리고 메서드 사이에는 한칸씩 개행해주면 더 가독성이 좋을것 같습니다